root/static/scripts/release-editor/edits.js

```diff
@@ -114,10 +114,10 @@ releaseEditor.edits = {
     var edits = [];
 
     for (let newLabel of newLabels) {
-      const id = newLabel.release_label;
+      const id = getReleaseLabel(newLabel);
```

Comment:
One thing to note with `Map` is that `1` (the number) and `"1"` (the string) are two different keys.  With regular objects these would be the same key.  So I'm using the same function that was passed to `keyBy` here to get the (string) id for lookup.